eng/release/Scripts/InstallAzCopy.ps1

```diff
@@ -0,0 +1,48 @@
+[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$true)][string] $ToolsDirectory,
+    [Parameter(Mandatory=$false)][string] $TaskVariableName = $null
+)
+
+$ErrorActionPreference = 'Stop'
+Set-StrictMode -Version 2.0
+
+$url = 'https://aka.ms/downloadazcopy-v10-windows'
+$basePath = Join-Path $ToolsDirectory 'azcopy'
```

Comment:
The old release definitions had a task to install azcopy. I'm a little surprised we have to download it ourselves.

Reply:
I couldn't find the task when mocking some of this up using the editor.

Reply:
Also, not sure if I would be able to manually figure out what the path to the azcopy executable would be if using that task.